lib/gui/lvgl/Kconfig

```diff
@@ -21,6 +22,14 @@ config APP_LINK_WITH_LVGL
 	  disabled if the include paths for LVGL are causing aliasing
 	  issues for 'app'.
 
+config LVGL_USER_CONFIG_FILE
```

Comment:
Suggestion: `LVGL_USER_CONFIG_HEADER` or `LVGL_USER_CONFIG_INCLUDE` would make it more clear as to what the "config file" is supposed to be.

Reply:
it would probably be correct to name  LVGL_USE_USER_CONFIG ?

Reply:
That does sound more reasonable.

`LVGL_USE_USER_CONFIG`, `LVGL_INCLUDE_USER_CONFIG_HEADER`, etc.